WalletWasabi/Logging/Logger.cs

```diff
@@ -350,6 +358,14 @@ public static void LogSoftwareStopped(string appName, [CallerFilePath] string ca
 		/// </summary>
 		public static void LogError(string message, [CallerFilePath] string callerFilePath = "", [CallerLineNumber] int callerLineNumber = -1) => Log(LogLevel.Error, message, callerFilePath: callerFilePath, callerLineNumber: callerLineNumber);
 
+		/// <summary>
+		/// Logs user message with exception concatenated to it at Error level.
+		///
+		/// For errors and exceptions that cannot be handled.
+		/// These messages indicate a failure in the current activity or operation (such as the current HTTP request), not an application-wide failure.
+		/// Example log message: "Cannot insert record due to duplicate key violation."
+		/// </summary>
+		public static void LogError(string message, Exception ex, [CallerFilePath] string callerFilePath = "", [CallerLineNumber] int callerLineNumber = -1) => Log(message, ex, LogLevel.Error, callerFilePath: callerFilePath, callerLineNumber: callerLineNumber);
```

Comment:
Can we put this in a new line?
```suggestion
		public static void LogError(string message, Exception ex, [CallerFilePath] string callerFilePath = "", [CallerLineNumber] int callerLineNumber = -1)
			=> Log(message, ex, LogLevel.Error, callerFilePath: callerFilePath, callerLineNumber: callerLineNumber);

```